units/patch_intent_test.go

```diff
@@ -679,3 +679,32 @@ func (s *PatchIntentUnitsSuite) TestCliBackport() {
 	s.Len(backportPatch.Patches, 1)
 	s.Equal(sourcePatch.Patches[0].PatchSet.Patch, backportPatch.Patches[0].PatchSet.Patch)
 }
+
+func (s *PatchIntentUnitsSuite) TestCantFinalizeChildPatch() {
```

Comment:
what is this testing?

Reply:
I'm moving this to patch_lifecycle so I'll be removing it.

Reply:
I was trying to test that it finishes without erring when there parent's status is not "", but I'm guessing that doesn't do much. I removed the test.